docs/DeveloperGuide.adoc

```diff
@@ -358,6 +358,16 @@ image::UndoRedoActivityDiagram.png[width="650"]
 ** Cons: Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two * different things.
 // end::undoredo[]
 
+===== Aspect: Support for multiple themes
```

Comment:
This is a subsection of the Undo/Redo feature. It should be part of a new section describing the implementation of multiple themes.